deps/libgit2.mk

```diff
@@ -40,6 +40,10 @@ ifeq ($(OS),Linux)
 LIBGIT2_OPTS += -DUSE_OPENSSL=OFF -DUSE_MBEDTLS=ON -DCMAKE_INSTALL_RPATH="\$$ORIGIN"
 endif
 
+ifeq ($(OS),FreeBSD)
+LIBGIT2_OPTS += -DCMAKE_INSTALL_RPATH="\$$ORIGIN"
```

Comment:
Yep, they're in `/etc/ssl/cert.pem`, same as what the comment says it is for Ubuntu. It's actually a symlink to `/usr/local/share/certs/ca-root-nss.crt`.